src/types/card.interface.ts

```diff
@@ -0,0 +1,63 @@
+interface card {
+  id: number;
+  list: Array<cardData>;
+}
+
+interface cardData {
+  id: string | number;
+  cardCVC: string;
+  cardNickname: string;
+  cardNumbers: cardNumbers;
+  cardPassword: string;
+  cardType: cardType;
+  expireDate: expireDate;
+  ownerName: ownerName;
+}
+
+interface cardType {
+  name: string;
+  color: string;
+}
+
+interface cardNumbers {
+  firstNumber: string;
+  secondNumber: string;
+  thirdNumber: string;
+  fourthNumber: string;
+}
+
+interface cardNumberInputObject {
+  name: cardNumberInputName;
+  type: "text" | "password";
+}
+
+interface expireDate {
+  month: string;
+  year: string;
+}
```

Comment:
간지나서 제안함22
ㅋㅋㅋㅋ근데 Record 로 객체 정의하면 더 확실하게 타입핑한다고..어디서 주워듣긴 했습니다..
아마도 객체의 key 값 타입도 정의할 수 있어서 아닐까하는 추측
```suggestion
type expireDate = Record<'month' | 'year' , string>
```

Reply:
좀 더 모던한 방법인 Record를 애용합시다! 🐱

- [refactor: 객체 프로퍼티 타입을 Record로 지정하도록 변경](https://github.com/usageness/react-payments/pull/1/commits/282874eb99cff72799dda7def01994478a45fcbd)